scalafix-tests/input/src/main/scala/test/DottyAutoTuplingFunctionArgs.scala

```diff
@@ -0,0 +1,40 @@
+/*
+rules = DottyAutoTuplingFunctionArgs
+ */
+package test
+
+object DottyAutoTuplingFunctionArgs {
```

Comment:
I would also add:
```scala
def foo[T](xs: List[T]) = xs.map { case (x, y) => 1 }
```

https://github.com/scalacenter/scalafix/issues/521#issuecomment-353063569

```scala
List(1, 2, 3).zipWithIndex.drop(1).take(1).reverse.map { case (x, y) => 1 }
  {
    if (Random.nextInt() > 0) Option(("a", "a"))
    else Option(("b", "b"))
  }.map { case (a, b) => a + b }
class Foo[T](val a: T)
implicit def foo2Opt[T](f: Foo[T]): Option[(T, T)] = Some((f.a, f.a))
new Foo(1).map { case (x, y) => 1 }
```

https://github.com/scalacenter/scalafix/issues/521#issuecomment-353363529

Reply:
The first one is already covered the bottom of the file, I don't see what new condition the other cases are supposed to cover.

Reply:
yeah, the other cases have no importance given the solution that Olafur has implemented.